scenarios/reproducers/va-hci.yml

```diff
@@ -64,6 +64,7 @@ cifmw_test_operator_tempest_include_list: |
 # provides access to Internet. This will be the equivalent of the
 # "public network" as seen in CI.
 cifmw_use_libvirt: true
+cifmw_virtualbmc_daemon_port: 50881
```

Comment:
setting this one here, since virtualbmc is deployed "by us" in any cases, and this one conflicts with the dev-scripts managed one.
We may consider modifying the `virtualbmc` role to allow consuming the dev-scripts managed one "when available" - though it's running as root (don't know why, will investigate the actual reasons) with privileges (no idea why, it seems to work just fine without it)....